src/controllers/ExamController.ts

```diff
@@ -0,0 +1,74 @@
+import { Request, Response} from 'express'
+import Exam from '../models/Exam'
+
+class ExamController{
+    static async getAllExams(req: Request, res: Response){
+        try{
+            const exams = await Exam.find().populate('course').populate('student')
+            res.json(exams)
+        }catch(error){
+            console.log(error)
+        }
+    }
+
+    static async getExamById(req: Request, res: Response){
+        const ExamId = req.params.id
+        try{
+            const exam = await Exam.findById(ExamId).populate('course').populate('student')
+            if(!exam){
+                return res.status(404).json({ message: 'Exam not found'})
+            }
+            res.json(exam)
+        }catch(error){
+            console.log(error)
```

Comment:
please replace all console.log with log4js library (replace it with logger.info('...'), logger.error('....'))